certified-connectors/DocuSignDemo/script.csx

```diff
@@ -1047,40 +1047,73 @@ public class Script : ScriptBase
 
     if (verificationType.Equals("Phone Authentication"))
     {
-      var identityVerification = new JObject();
-      var inputOptions = new JArray();
-      var inputObject = new JObject();
-      var phoneNumberList = new JArray();
-      var phoneNumberObject = new JObject();
+      try
+      {
+        var identityVerification = new JObject();
+        var inputOptions = new JArray();
+        var inputObject = new JObject();
+        var phoneNumberList = new JArray();
+        var phoneNumberObject = new JObject();
+
+        phoneNumberObject["Number"] = body["phoneNumber"];
+        phoneNumberObject["CountryCode"] = body["countryCode"];
+        phoneNumberList.Add(phoneNumberObject);
 
-      phoneNumberObject["Number"] = body["phoneNumber"];
-      phoneNumberObject["CountryCode"] = body["countryCode"];
-      phoneNumberList.Add(phoneNumberObject);
+        inputObject["phoneNumberList"] = phoneNumberList;
+        inputObject["name"] = "phone_number_list";
+        inputObject["valueType"] = "PhoneNumberList";
+        inputOptions.Add(inputObject);
 
-      inputObject["phoneNumberList"] = phoneNumberList;
-      inputObject["name"] = "phone_number_list";
-      inputObject["valueType"] = "PhoneNumberList";
-      inputOptions.Add(inputObject);
+        identityVerification["workflowId"] = body["workflowID"];
+        identityVerification["inputOptions"] = inputOptions;
+        recipient["identityVerification"] = identityVerification;
 
-      identityVerification["workflowId"] = body["workflowID"];
-      identityVerification["inputOptions"] = inputOptions;
-      recipient["identityVerification"] = identityVerification;
+        if (string.IsNullOrEmpty((body["phoneNumber"]).ToString()) || string.IsNullOrEmpty((body["countryCode"]).ToString()) || string.IsNullOrEmpty((body["workflowID"]).ToString()))
+        {
+          throw new Exception();
```

Comment:
Instead of throwing an exception without any details, can you add a message such as
"phone number, country code or workflow id information is missing"